### ISSUE TITLE
How to evaluate a new Image using CNTK FAST R-CNN

### ISSUE THREAD

**Issue by gsrungarapu** [Microsoft/CNTK#1888]
I ran the Grocery dataset (cntk + fast r-cnn) tutorial and was successfully able to train and test on the images. 

If we have to train a big dataset,  we are creating annotations and labels manually (for both train and test sets). 

However, If we have a new image how do we apply the trained model and evaluate the image to detect the objects in it without we manually annotate/apply labels. Is that possible?

**Comment by DHOFM** [comment]
Hi,
Nadav made a code commit, that can be used for that. You can read my discussion with him, regarding this issue: https://github.com/CatalystCode/CNTK-FastRCNNDetector/issues/1 But with self labeled training data you should not have any problems using his code. 

Kind regards,

Dirk

**Comment by gsrungarapu** [comment]
That is great! Thanks @DHOFM  
I will try this code. However, as I understand for a new image to do evaluation on it, so we still need to select the ROI's manually or adjust the parameters in the code, such that the ROI's and labels will be automatically identified and produced?

Thanks!

**Comment by DHOFM** [comment]
@gsrungarapu: For a new image the ROI Selection should run. I had a problem with the Pascal Demo Dataset, because in that case, the parameters and labels are loaded from the Matlab Files i mentioned. In your case, with your own dataset, you know the parameters for your input rois while training. So with your parameters the code should work. If you use a GPU it can be used in the evaluation. Even with not knowing the training ROI parameters i get some good results now, by testing the parameters. In that case you can plot the rois in the image, witch can help alot.

Kind regards,

Dirk.
